definitions/axios_v0.11.x/test_axios-v0.11.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import axios from 'axios';
+(axios.get('foo'): Promise);
```

Comment:
Would you mind specifying `Promise<*>` here instead?

In 0.25, Flow is going to start requiring that parameterized types have all of their parameters explicitly instantiated. Using `*` just means "I explicitly want Flow to infer the type"